frontend/src/libs/components/stepper/step-target/stepper-target.tsx

```diff
@@ -0,0 +1,46 @@
+import { useForm } from 'react-hook-form';
+
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+
+import { SelectTarget } from '../../select-target/select.js';
```

Comment:
Should use default select with enhanced stuff
```suggestion
import { Select } from '~/libs/components/components.js';
```